lightning/src/onion_message/mod.rs

```diff
@@ -0,0 +1,33 @@
+// This file is Copyright its original authors, visible in version control
+// history.
+//
+// This file is licensed under the Apache License, Version 2.0 <LICENSE-APACHE
+// or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// You may not use this file except in accordance with one or both of these
+// licenses.
+
+//! Onion Messages: sending, receiving, forwarding, and ancillary utilities live here
+//!
+//! Onion messages are multi-purpose messages sent between peers over the lightning network. In the
+//! near future, they will be used to communicate invoices for [offers], unlocking use cases such as
+//! static invoices, refunds and proof of payer. Further, you will be able to accept payments
+//! without revealing your node id through the use of [blinded routes].
```

Comment:
Consider writing this without qualifiers like "In the near future" and without using future tense given the crate is not public yet. Likewise in `messenger.rs`.

Reply:
Hmm, even when the crate is public, offers still won't be supported. IIUC, they'd be inaccurate-for-the-current-state for a few months if written in the present tense?

Reply:
Gotcha. I guess I'd consider writing in the present tense and adding a note saying it is not yet implemented, which could be easily removed later. Your call.

Reply:
I think these may need some augmenting/rewriting for offers anyway, gonna leave for now